tests/test_core.py

```diff
@@ -3162,7 +3162,7 @@ def post(filename):
       if self.is_wasm_backend():
         self.assertLess(len(exports), 56)
       else:
-        self.assertLess(len(exports), 32)
+        self.assertLess(len(exports), 33)
```

Comment:
This is because we are including the libc-extras library.  This library contains a extra global called `environ`   .. its in a separate library so that its only included on demand.  However when building with MAIN_MODULE=1 the ideas is that we include all library function and don't do gc. 

So before I would argue it was a bug that we were not including environ in this case.